src/explorer/installation/installation-explorer.ts

```diff
@@ -39,7 +39,7 @@ export class InstallationExplorer implements vscode.TreeDataProvider<Installatio
     }
 
     refresh(): CommandResult {
-        this.onDidChangeEmitter.fire();
+        this.onDidChangeEmitter;
```

Comment:
Does this line do anything? Previously it was triggering the event handler.

Reply:
💡🥷 🧑‍🔧 Thank you for asking this - Oh, let me check on this, there must be a reason, might be because this function previous use to take no-parameter and now takes `data` as param and the whole extension `set-up` was little interesting in case of how the `TreeNodeObject` in this scenario will be passed. 

<img width="876" alt="Screenshot 2023-01-09 at 9 51 06 PM" src="https://user-images.githubusercontent.com/6233295/211359653-5a700f00-1128-457b-a977-dd973e733d01.png">

Reply:
☕️💡 I have tried to pass right object what I can see, will it be cool if you can please test this, or I can share six, but definitely need another eyes.